nuntium/templates/nuntium/create_new_writeitinstance.html

```diff
@@ -13,38 +13,71 @@
 politicians.</p>
-
-<p>You will need:</p>
-
-<ul>
-  <li> Names and contact details for those politicians</li>
-  <li> A <a href="http://popit.poplus.org/">PopIt</a> site to store them in</li>
-</ul>
-
-<p>Both sites work hand in hand: PopIt stores your data, and we fetch the
-names and contact details we need from there.</p>
-
-<h4>No PopIt site?</h4>
-
-Don’t worry, it takes just minutes to set one up. All you need is a spreadsheet with the basic data for your politicians. 
-
-The <a href="http://everypolitician.org/upload/">CSV importer at EveryPolitician</a> 
-will turn that data into the right format, and create a PopIt for you.
-
-<h4>Got your PopIt?</h4>
-
-<p>Great - you’re ready to begin!</p>
-
-<hr>
-
 {% endblocktrans %}
-
 <form method="POST">
+  <div class="form-group">
+    <label>Legislature</label>
+    <select class="form-control js-countries-list">
+      <option>Select a legislature</option>
+      <option disabled></option>
+      {% for country in countries %}
+      <option
+      data-slug="{{ country.slug }}"
+      data-email-count="{{ country.people_with_contact_details }}"
+      data-person-count="{{ country.person_count }}"
+      value="{{ country.url }}"
+      >{{ country.country }} - {{ country.legislature }}</option>
+      {% endfor %}
+    </select>
+    <span class="helptext js-not-enough-contact-details">No contact details</span>
+    <span class="helptext js-low-contact-count-warning"></span>
+  </div>
+  <p><a href="#" class="js-show-manual-popolo">I have my own suitably formatted popolo</a></p>
     {% csrf_token %}
     {{form.as_p}}
-    <button class='btn btn-primary'>{% trans 'Create' %}</button>
+    <button class='btn btn-primary js-submit-button'>{% trans 'Create' %}</button>
 </form>
 
+<script>
+  var notEnoughContactDetails = $('.js-not-enough-contact-details');
+  notEnoughContactDetails.hide();
+  var popitUrl = $('#id_popit_url').closest('p');
+  popitUrl.hide();
+  var countriesList = $('.js-countries-list');
+  var submitButton = $('.js-submit-button');
+  var lowContactCountWarning = $('.js-low-contact-count-warning');
+  lowContactCountWarning.hide();
+  submitButton.prop('disabled', true);
+  $('.js-show-manual-popolo').click(function(e) {
+    e.preventDefault();
+    popitUrl.show();
+    countriesList.closest('div').hide();
+    $(this).hide();
+  });
+  countriesList.change(function(e) {
+    notEnoughContactDetails.hide();
+    lowContactCountWarning.hide();
+    var option = $('option:selected', this);
+    var slug = option.data('slug');
+    var emailCount = option.data('email-count');
+    var personCount = option.data('person-count');
+    if (emailCount > 0) {
+      submitButton.removeAttr('disabled');
+      $('#id_popit_url').val($(this).val());
+      $('#id_slug').val(slug);
+      if ((emailCount / personCount) <= 0.5) {
+        var text = 'We only have email addresses for ' + emailCount + ' out of ' + personCount + ' people.';
```

Comment:
My only concern is here where it seems that there is no translation for this javascript text. Anyhow this is not something for which I should not merge this into master.